lib/registration.pm

```diff
@@ -789,13 +789,13 @@ sub install_docker_when_needed {
     }
     else {
         if (is_sle('<15')) {
-            assert_script_run('zypper se docker || zypper -n ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/SUSE:SLE-12:Update.repo');
+            script_run('zypper -n ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/SUSE:SLE-12:Update.repo');
```

Comment:
I doubt it's good to ignore failures here, I recommend to keep the assert_script_run

Reply:
Good point, now it's back, just with increesed timeout. The timeout was too low and the test was failing over there.